src/Features/Core/GenerateType/AbstractGenerateTypeService.Editor.cs

```diff
@@ -328,7 +328,8 @@ private void AddFoldersToNamespaceContainers(List<string> container, IList<strin
                     documentId,
                     documentName,
                     containers,
-                    sourceCodeKind));
+                    sourceCodeKind,
+                    filePath: Path.Combine (Path.GetDirectoryName (generatingDocument.FilePath), documentName)));
```

Comment:
That's the years of the mono formatting :/

I'm not sure if it's an issue for VS.NET or not.